### PR TITLE
[Program: GCI] Add potential error responses to Swagger UI (Backend)

### LINKED ISSUE #345: Add potential error responses to Swagger UI
## Description
As a developer that uses the Swagger UI,
I need to know the potential error responses from the backend,
so that I can understand how to properly use the API.

Swagger user interface, the one accessible locally through localhost:5000, does not have all the error responses properly documented. You can see the 200 HTTP code, documented as a successful message, but some APIs may return 401, 403, 404 and these are not documented.
You can use annotations by from flask-restplus framework that allows you to do this.

### Update [Required]
- [ ] Document every HTTP error code response, by exposing them on the Swagger UI per each API (e.g.: 404 for user not found when sending a new request).

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by asawesome07**
### Description
added error responses to swagger

Fixes #345 

### Type of Change:
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
It was tested on local backend. Here's a screenshot of how it looks
c2

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -246,6 +280,10 @@ class LoginUser(Resource):
     @classmethod
     @users_ns.doc('login')
     @users_ns.response(200, 'Successful login', login_response_body_model)
```
Comment (sumit-badsara) [inline app/api/resources/user.py]: You can add these to `messages.py` and include here similarly like the other responses i.e `messages.LOGIN_SUCCESS['message']`
Check in other files too where it is hardcoded.

**Review by sumit-badsara — CHANGES_REQUESTED** [review]
Do these changes to all the relevant messages and files. :)

File: app/api/resources/user.py (hunk 1 above)
Comment (asawesome07) [reply app/api/resources/user.py]: done

File: app/messages.py [hunk 2]
```diff
@@ -203,6 +203,12 @@
 USER_SUCCESSFULLY_UPDATED = {"message": "User was updated successfully."}
 PASSWORD_SUCCESSFULLY_UPDATED = {"message": "Password was updated "
                                  "successfully"}
+SUCCESSFUL_LOGIN = {"message": 'Successful login'}
+SUCCESSFUL_RESPONSE = {"message": "Successful response"}
+SUCCESSFUL_REFRESH = {"message": 'Successful refresh'}
+RETURNED_USERS_SUCCESSFULLY = {"message": 'returned users successfully'}
+LIST_TASKS_FROM_MENTORSHIP_SUCCESSFULLY = {"message": 'List tasks from a mentorship relation with success.'}
+RETURN_ALL_USER_MENTORSHIP_RELATIONS_SUCCESSFULLY = {"message": 'Return all user\'s mentorship relations was successfully.'}
```
Comment (sumit-badsara) [inline app/messages.py]: was successfully?
returned successfully*

return
Returned*

**Review by sumit-badsara — CHANGES_REQUESTED** [review]
Minor correction. And maintain a bit of consistency of which type of quotes you use. Either single or double. Code looks a bit clean and better.

File: app/messages.py [hunk 3]
```diff
@@ -203,6 +203,12 @@
 USER_SUCCESSFULLY_UPDATED = {"message": "User was updated successfully."}
 PASSWORD_SUCCESSFULLY_UPDATED = {"message": "Password was updated "
                                  "successfully"}
+SUCCESSFUL_LOGIN = {"message": "Successful login"}
+SUCCESSFUL_RESPONSE = {"message": "Successful response"}
+SUCCESSFUL_REFRESH = {"message": "Successful refresh"}
+RETURNED_USERS_SUCCESSFULLY = {"message": "returned users successfully"}
+LIST_TASKS_FROM_MENTORSHIP_SUCCESSFULLY = {"message": "List tasks from a mentorship relation with success."}
+RETURNED_ALL_USER_MENTORSHIP_RELATIONS_SUCCESSFULLY = {"message": "Returned all user\'s mentorship relations was successfully."}
```
Comment (sumit-badsara) [inline app/messages.py]: Change this to
```
Successfully returned all mentorship relations of the user.
```

**Review by sumit-badsara — COMMENTED** [review]
Replace with the given message. Everything else seems fine.

**Comment by isabelcosta** [comment]
@asawesome07 thank you so much for your contribution! I'm closing this PR because we decided to merge PR #365

**Pull request closed without merging**